modules/frontend/tracebyidsharding.go

```diff
@@ -168,8 +171,12 @@ func (s shardQuery) RoundTrip(r *http.Request) (*http.Response, error) {
 		switch statusCode {
 		case http.StatusNotFound:
 			// Pass through 404s
+		case http.StatusRequestTimeout:
+			// Pass through 408s
 		case http.StatusTooManyRequests:
 			// Pass through 429s
+		case 499:
```

Comment:
similar thought here? have we seen 499s? do we know if the go http client actually returns this status code on timeout? i would expect it just to return errors.

Reply:
See the response above.